webview-ui/src/components/chat/ErrorRow.tsx

```diff
@@ -117,21 +170,36 @@ export const ErrorRow = memo(
 
 		// Standard error display
 		return (
-			<>
+			<div className="group pr-2">
 				{errorTitle && (
-					<div className={headerClassName || "flex items-center gap-2 break-words"}>
+					<div className={headerClassName || "flex items-center justify-between gap-2 break-words"}>
 						<MessageCircleWarning className="w-4 text-vscode-errorForeground" />
-						<span className="text-vscode-errorForeground font-bold">{errorTitle}</span>
+						<span className="text-vscode-errorForeground font-bold grow cursor-default">{errorTitle}</span>
+						{docsURL && (
+							<a
+								href={docsURL}
+								className="text-sm flex items-center gap-1 transition-opacity opacity-0 group-hover:opacity-100"
+								onClick={(e) => {
+									e.preventDefault()
+									vscode.postMessage({ type: "openExternal", url: docsURL })
+								}}>
+								<BookOpenText className="size-3" />
+								Docs
```

Comment:
Should we internationalize "Docs"?

Reply:
Maybe the Improover can update the rules once this is merged to make it more likely for code review to catch this 🙏